pdns/dnspacket.cc

```diff
@@ -325,6 +335,11 @@ void DNSPacket::wrapup()
         opts.push_back(make_pair(8, opt)); // 'EDNS SUBNET'
       }
 
+      if (d_haveednscookie && d_eco.isWellFormed()) {
+        d_eco.makeServerCookie(s_EDNSCookieKey, getRemote());
```

Comment:
@Habbie Should this become `getInnerRemote`?

Reply:
after PROXY is merged, yes!